samples/ProjectGrandmaster.Unity/Assets/Editor/DebugPosition.cs

```diff
@@ -0,0 +1,14 @@
+﻿using UnityEngine;
+using UnityEditor;
+
+public static class DebugPosition
+{
+    [MenuItem("Debug/Print Global Position")]
+    public static void PrintGlobalPosition()
+    {
+        if (Selection.activeGameObject != null)
+        {
+            Debug.Log(Selection.activeGameObject.name + " is at " + Selection.activeGameObject.transform.position);
```

Comment:
Could you look this up by selecting the object and looking at the inspector?

Reply:
Did not make use of this script. Can be deleted.